### ISSUE TITLE
[critical][www.etsy.com]

### ISSUE THREAD

**Issue by ckundo** [AccessLint/reports#8]
## Snippet

Ensures elements with an ARIA role that require parent roles are contained by them

```html
<span id="catnav-primary-link-10983" class="wt-text-black" role="menuitem" aria-haspopup="true" tabindex="0">
                           Wedding &amp; Party
                       </span>
```

![Ensures elements with an ARIA role that require parent roles are contained by them](https://raw.githubusercontent.com/AccessLint/reports/main/4646e343196c3e281183c667b8f8830b49e1d879af5a966f03cec231d95ec82c.png) 

engine: [axe-core](https://github.com/dequelabs/axe-core)
